modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -479,3 +481,70 @@ test('can import custom object entries at instance level with or without permiss
 		})
 	);
 });
+
+test('can see corresponding elements in instance and site level', async ({
+	apiHelpers,
+	companyExportImportPage,
+	exportImportPage,
+}) => {
+	const objectActionApiClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionApi);
+
+	const {body: objectDefinition} =
+		await objectActionApiClient.postObjectDefinition({
+			active: true,
+			externalReferenceCode: 'test',
+			label: {
+				en_US: 'Test',
+			},
+			name: 'Test',
+			objectFields: [
+				{
+					DBType: ObjectField.DBTypeEnum.String,
+					businessType: ObjectField.BusinessTypeEnum.Text,
+					indexed: true,
+					indexedAsKeyword: true,
+					label: {
+						en_US: 'Name',
+					},
+					name: 'name',
+					required: true,
+				},
+			],
+			pluralLabel: {
+				en_US: 'Tests',
+			},
+			portlet: true,
+			scope: 'company',
+			status: {
+				code: 0,
+			},
+		});
+
+	apiHelpers.data.push({id: objectDefinition.id, type: 'objectDefinition'});
+
+	const exportFilePath =
```

Comment:
@4lejandrito should we split these tests? One for site and the other one for instance?